server/routers/_app.ts

```diff
@@ -16,7 +18,7 @@ export const appRouter = createRouter()
    * Add data transformers
    * @link https://trpc.io/docs/data-transformers
    */
-  // .transformer(superjson)
+  .transformer(superjson)
```

Comment:
You don't seem to be using trpc on this PR so maybe we should leave this for a PR that actually will need it?

Reply:
@KATT It's just some prep-work for the next PRs